versions.props

```diff
@@ -11,7 +11,7 @@ org.apache.spark:spark-avro_2.11 = 2.4.8
 org.apache.pig:pig = 0.14.0
 com.fasterxml.jackson.*:* = 2.11.4
 com.google.errorprone:error_prone_annotations = 2.3.3
-com.google.guava:guava = 28.0-jre
+com.google.guava:guava = 31.0.1-jre
```

Comment:
@aokolnychyi, @RussellSpitzer, @jackye1995, any thoughts on upgrading Guava? It should be safe because we shade and relocate. Not sure if there have been behavior changes, though.

Reply:
We shade it so I have no problems with the upgrade. May want to do it after release though just in case

Reply:
Agreed. We should do this after the release.

Reply:
+1